lib/MusicBrainz/Server/Form/OtherLookup.pm

```diff
@@ -28,19 +28,19 @@ has_field 'iswc'     => (
     type => '+MusicBrainz::Server::Form::Field::ISWC',
 );
 
-has_field 'artist-ipi'     => (
+has_field 'artist_ipi'     => (
```

Comment:
That breaks all existing other lookup URLs with such parameter.

Reply:
Fair point. Dunno how many we have but I guess I'll just do field['artist-ipi'] and cry a bit :D